src/ui.form-checkbox-group/index.stories.js

```diff
@@ -67,14 +67,14 @@ const EnumVariantTemplate = (args, { argTypes } = {}) => ({
   template: `
     <URow class="!flex-col">
       <UCheckboxGroup
-        v-bind="args"
         v-for="(option, index) in options"
         :key="option"
+        v-model="value"
```

Comment:
I'm so sorry. `v-model` should be after `v-bind` in this case it doesn't shows on change in args.